### ISSUE TITLE
Dependency with Alexa "Notify Me" skill 

### ISSUE THREAD

**Issue by jimbotel** [bartbutenaers/node-red-contrib-alexa-notifyme#5]
In the README in GitHub (as well as in node-red page https://flows.nodered.org/node/node-red-contrib-alexa-notifyme) you should explicitly mention that this node requires enabling  "Notify Me" skill in Alexa in order to work. 
Note that "Notify Me"is only available in English AND in certain countries (for example, Amazon does not allow you to enable that skill in Spain or in Germany, even if you didn't mind having it in English).

**Comment by juggledad** [comment]
Good point, I'll make sure that it is updated. Although in order to get your access code, it does give a link to get it. That link takes you to the Notify Me site which does tell you to enable the skill so it will send you the access code:

> To get started, just enable the skill, giving it permission to send notifications to your Alexa and linking it to your Amazon account so we can send a unique access code to the email address associated with that account.  Next, just say, "Alexa, open Notify Me" and the skill will introduce itself and send your access code via email.

BTW - did you leave them feedback asking about support for other languages?

**Comment by jimbotel** [comment]
Hi, 
Thanks for the prompt answer. I saw the link, yes, but by the time you get there most probably you have already installed the node and are very excited about making it work, and then you get the "bluff" moment :-(.

I did give feedback to Notify Me developers. Here's the reply (from today):

------------------

Thanks for your interest in Notify Me.

Unfortunately, we lack the resources to translate the skill to other languages.  As you might know, we don't make any money for developing or maintaining the skill.  Even if all the text was translated for us, someone would have to insert the translations into the code, test the results, manage it through the submission process, and then pay for the additional server usage while also providing ongoing support.

As a result, Notify Me is only available in the countries that allow English-speaking skills.

------------------

So it does not seem to be "in plan". 

Regards,
Jaime

**Comment by juggledad** [comment]
Well seeing that I'd better go back and correct the changes to the README that I just made. Thank You for providing that feedback!

**Comment by jimbotel** [comment]
Thanks, I've seen the updates. I hope  Notify Me or a similar skill will be made available in Spanish.
